WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerActivity.kt

```diff
@@ -190,55 +190,54 @@ class MediaPickerActivity : LocaleAwareActivity(), MediaPickerListener {
                     return
                 }
             }
-            TAKE_PHOTO -> {
-                try {
-                    val intent = Intent()
-                    mediaCapturePath!!.let {
-                        WPMediaUtils.scanMediaFile(this, it)
-                        val f = File(it)
-                        val capturedImageUri = listOf(Uri.fromFile(f))
-                        if (mediaPickerSetup.queueResults) {
-                            intent.putQueuedUris(capturedImageUri)
-                        } else {
-                            intent.putUris(capturedImageUri)
-                        }
-                        intent.putExtra(
-                                EXTRA_MEDIA_SOURCE,
-                                ANDROID_CAMERA.name
-                        )
-                    }
-                    intent
-                } catch (e: RuntimeException) {
-                    AppLog.e(MEDIA, e)
-                    null
-                }
-            }
-            IMAGE_EDITOR_EDIT_IMAGE -> {
-                data?.let {
-                    val intent = Intent()
-                    val uris = WPMediaUtils.retrieveImageEditorResult(data)
-                    if (mediaPickerSetup.queueResults) {
-                        intent.putQueuedUris(uris)
-                    } else {
-                        intent.putUris(uris)
-                    }
-                    intent.putExtra(
-                            EXTRA_MEDIA_SOURCE,
-                            APP_PICKER.name
-                    )
-                    intent
-                }
-            }
-            else -> {
-                data
-            }
+            TAKE_PHOTO -> takeAPhoto()
+            IMAGE_EDITOR_EDIT_IMAGE -> data?.let {  editImageIntent(it) }
+            else -> data
+
```

Comment:
Minor (🔍): Consider removing the extra empty line here.

Reply:
@ParaskP7  fix this lint error, also think i should mention that i am usually unable to see whatever you're trying to show me with the `buildkite` link 🫣, probably some set up i haven't done 👀

Reply:
👋 @Hoossayn and thank you for this fix! 👍

> ...think i should mention that i am usually unable to see whatever you're trying to show me with the buildkite link 🫣, probably some set up i haven't done 👀

Ah, probably you don't (and maybe can't) access `Buildkite` on your side, I'll confirm that and get back to you.

As such, and until then, I suggest you do the following, observe the CI checks and see what is failing, then try to run in on your side to observe the failure locally as well. For example, when you see `CheckStyle` failing you could run `./gradlew checkstyle` locally and see it's output. You can also check the [.buildkit/pipeline.yml](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/.buildkite/pipeline.yml) file for all the other checks and how you could trigger them locally.

Another tip I can recommend for you to be doing before pushing any code, is to at least run these check locally first:
- CheckStyle: `./gradlew checkstyle`
- Detekt: `./gradlew detekt`
- Lint: `./gradlew lintWordPressWasabiDebug`
- Test: `./gradlew testWordPressWasabiDebugUnitTest`

The above local checks will get you to almost 99% confidence, that is, in terms of the fact that our CI checks will also succeed afterwards and thus help you avoid this ping-pong that is happening atm.

Let me know if the above is helpful to you. 🙏

Reply:
👋🏾  @ParaskP7  Above check builds successfully except the last one, which i checked against trunk and was falling too